File: Modul 5/Soal 5/PRAK405-2210817220018-Piola Evania.py
for i in range(3):
    a, b = input().split()
    a = int(a); b = int(b)

    for i in range(1, a + 1):
        j = i 
        while j > 1:
            print("(%d * %d) + "%(j, b), end=' ')
            j -= 1
        c = 1; hasil = (i * b)
        while c < i:
            hasil += (c * b); c += 1
        print("(%d * %d) = %d"%(j, b, hasil))
    p = 1; q = 1; r = 0
    while p <= a:
        r += q * b; q += p + 1; p += 1
    print("%d"%(r))
    print("\n")

File: Modul 5/Soal 5/PRAK405-2210817220018-Piola Evania.c
#include<stdio.h>

int main()
{
    for(int x = 1; x < 4; x++){
    int a, b, c, i, j, p, q, r, hasil;
    scanf("%d %d", &a, &b);
    for (i=1; i<=a; i++){
        for(j=i; j>1; j--){
            printf("(%d * %d) + ", j, b);
        }
        for(c=1, hasil=c*b; c<i; c++, hasil+=(c*b)){}
        printf("(%d * %d) = %d\n", j, b, hasil);
    }
    for (p=1, q=1, r = 0; p <= a; q += p + 1, p++){
        r += q * b;
    }
    printf("%d", r);
    printf("\n\n");
    }
}
